packages/office-ui-fabric-react/src/components/Layer/Layer.types.ts

```diff
@@ -68,6 +68,12 @@ export interface ILayerProps extends React.HTMLAttributes<HTMLDivElement | Layer
    * By default, the layer will be appended at the end to the host
    */
   insertFirst?: boolean;
+
+  /**
+   * Whether to render the layer in the top-most DOM document. If true, top-most will be used,
+   * if false, the current document will be used.
+   */
+  topMostDocument?: boolean;
```

Comment:
Related to other comment, I wonder if another name like `renderAtRoot` or `renderAtTop` may be more clear.

Reply:
See bdeeba7dfb672fd4b2db840b6d065a8a629d3953